client/src/main/java/io/pravega/client/tables/KeyValueTable.java

```diff
@@ -142,7 +142,7 @@
      * into this {@link KeyValueTable}. All changes are performed atomically (either all or none will be accepted).
      *
      * @param keyFamily The Key Family for the all provided Table Entries.
-     * @param entries   An {@link Iterable} of {@link Map.Entry} instances to insert or update.
+     * @param entries   An {@link Iterable} of {@code Map.Entry} instances to insert or update.
```

Comment:
This should be link, not code.

Reply:
The problem is that using `@link` here throws a warning (the same as in the other point in which this change is done). I suspect that is related to the visibility of that inner class.

Reply:
That is a public nested interface in the JDK. It's not a private class.

Reply:
Fixed by adding the complete name to the class.